File: LVJ_AdminVersion/ModeloPersistênciaPython/Categoria.py
class Categoria:
    def __init__(self, id, descricao, desconto):
        self._id = id
        self._descricao = descricao
        self._desconto = desconto


    @property
    def id(self):
        return self._id
    @id.setter
    def id(self, novo_id):
        self._id = novo_id

    @property
    def descricao(self):
        return self._descricao
    @descricao.setter
    def descricao(self, nova_descricao):
        self._descricao = nova_descricao

    @property
    def desconto(self):
        return self._desconto
    @desconto.setter
    def desconto(self, novo_desconto):
        self._desconto = novo_desconto
    
    def __str__(self):
        if self._desconto == 0:
            return f"{self._id} - {self._descricao}\n"
        else:
            return f"{self._id} - {self._descricao} - {self._desconto}% de Desconto \n"
    #     }
    # }

File: LVJ_AdminVersion/ModelosPersistencia/Categoria.cs
class Categoria
{
    public int id { get; set; }
    public string descricao { get; set; }
    public double desconto { get; set; }

    public Categoria(int id, string descricao, double desconto)
    {
        this.id = id;
        this.descricao = descricao;
        this.desconto = desconto;
    }

    public override string ToString()
    {
        if(desconto == 0)
        {
            return $"[{id}] - {descricao} \n";
        } else {
            return $"[{id}] - {descricao} - {desconto}% de Desconto \n";
        }
    }
}